### PR TITLE
CIE-4102. Added content for reserved IP addresses.

### PR DESCRIPTION
**Pull request by sharath-jk**

### CODE REVIEW

File: content/edge/configuration-bundle/configuring-the-network.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ layout: redirect
 After a successful installation, you can reconfigure the network and also configure the network CIDR.
 
 {{< c8y-admon-important >}}
-Do not use the IP addresses 10.244.0.0 and 10.96.0.0 in your network configuration. These IP addresses are reserved for {{< product-c8y-iot >}} Edge internal purpose.
+
+Do not use the below IP addresses in your network configuration. These IP addresses are reserved for {{< product-c8y-iot >}} Edge internal purpose.
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-the-network.md:11]: - [ ] Do not use the IP addresses from the below blocks in your network configuration ...

File: content/edge/configuration-bundle/configuring-the-network.md [hunk 2]
```diff
@@ -7,7 +7,12 @@ layout: redirect
...
+
+- 10.96.0.0/12 - reserved for Kubernetes service
+- 10.88.0.1/16 - reserved for Podman network
```
Comment (bbyreddy) [inline content/edge/configuration-bundle/configuring-the-network.md:14]: - [ ] How did we determine these address blocks?

File: content/edge/configuration-bundle/configuring-the-network.md (hunk 1 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-the-network.md:11]: Done.

**Review by rssag — APPROVED** [review]

**Review by bbyreddy — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-the-network.md (hunk 2 above)
Comment (sharath-jk) [reply content/edge/configuration-bundle/configuring-the-network.md:14]: Mukund and Chandra confirmed these address blocks.

**Pull request merged**